config/settings.yml

```diff
@@ -87,10 +87,17 @@ salesforce-carma:
   signing_key_path: "spec/fixtures/carma/privatekey.pem" # A dummy private key used for test suite
   username: fake-user@va.gov.carmafake
 
-salesforce:
-  env: dev
+salesforce-gibft:
+  url: "https://va--rdtcddev.cs33.my.salesforce.com/"
 
 form_10_10cg:
+  carma:
+    mulesoft:
+      mock: true
+      host: "https://fake-mulesoft.vapi.va.gov"
+      client_id: BEEFCAFE1234
```

Comment:
cute, but you might wan to stick to `mulesoft-client-id` or someone is going to spend time trying to figure out where the Beef was generated.